milo-rastgoo.txt

```diff
@@ -0,0 +1,103 @@
+Link to codepen assignment: https://codepen.io/rastgoo/pen/ZEEXWvG
+
+1. What is Semantic HTML? 
+semantic html introduces elements with meaning to web page layouts. it is used both for web browsers and user agents such as screen readers.
+2. What is HTML used for? 
+HTML is used to markup contents of a web page. Markup part is similar to traditional editorial tags/markings for newspapers/books editorials and Hypertext part of it refers to webpages with innerlinks to each other and other resources. 
+3. What is an attribute and where do we put it?
+Attributes provide additional information about an element's configuation or behaviour.An attribute is added to an HTML start tag. 
+4. What is the h1 tag used for? How many times should I use it on a page?
+h1 is the first level heading. General best practice is to use it only once on every webpage for the most important heading.
+5. Name two tags that have required attributes
+a, img
+6. What do we put in the head of our HTML document? 
+meta information that is targetted to web browsers, search engines etc. and not for human beings.
```

Comment:
There is one human readable element: `<title></title>`

Reply:
Aha, Thanks. I have to pay more attention